docs/topics/proc_preparing-nodejs-application-for-openshift-deployment.adoc

```diff
@@ -0,0 +1,70 @@
+[id='preparing-a-application-for-openshift-deplyoment_{context}']
+= Preparing {Node} application for OpenShift deployment
+
+To prepare a {Node} application for OpenShift deployment, you must perform the following steps:
+
+* Add `nodeshift` to the application.
+* Add `openshift` and `start` entries to the `package.json` file.  
+
+.Prerequisites
+* `npm` installed.
+
+.Procedure
+. Add `nodeshift` to your application.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install nodeshift --save
```

Comment:
we usually use --save-dev here since it isn't needed to actually run the application,  just deploy it